packages/terra-functional-testing/.npmignore

```diff
@@ -0,0 +1,4 @@
+*.log
+target
+reports
+src
```

Comment:
will probably need to add a few things here like config files, error screenshots, tests, etc. so they aren't released with the component

Reply:
My plan is to add those as they become necessary. Tests can probably be added now